testing/unittests/accessmaptests.cpp

```diff
@@ -0,0 +1,153 @@
+/*##############################################################################
+
+    Copyright (C) 2021 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifdef _USE_CPPUNIT
+#include "AccessMapGenerator.hpp"
+#include "EsdlAccessMapGenerator.hpp"
+#include "unittests.hpp"
+
+class AccessMapTests : public CppUnit::TestFixture
```

Comment:
Since the unittests directory is shared the name should probably reflect that these are EsdlAccressMapTests.

Reply:
Will change.